test/unittests/test_pla_parser.cpp

```diff
@@ -0,0 +1,80 @@
+#include "core/truthTable/pla_parser.hpp"
+
+#include "gtest/gtest.h"
+
+using namespace syrec;
+
+class PlaParserTest: public testing::TestWithParam<std::string> {
+protected:
+    std::string       test_circuits_dir = "./circuits/";
+    syrec::TruthTable testPla;
+};
+
+TEST_F(PlaParserTest, andTest) {
+    std::string circAnd = test_circuits_dir + "and.pla";
+
+    EXPECT_EQ(testPla.nInputs(), 0U);
+    EXPECT_EQ(testPla.nOutputs(), 0U);
+    EXPECT_EQ(testPla.ioCube().size(), 0U);
```

Comment:
these checks are superfluous and can be removed in all of the tests as they will always be satisfied and test nothing.